pkg/domain/entities/generate.go

```diff
@@ -32,6 +32,8 @@ type GenerateSystemdOptions struct {
 	After []string
 	// Requires - systemd requires list for the container or pods
 	Requires []string
+	// Envs - environment variables setted by -e/--env
```

Comment:
```suggestion
	// Envs - environment variables set by -e/--env
```

Reply:
I suggest removing the reference on the CLI flag as it's not important in/for this package.